src/formats/newick.rs

```diff
@@ -44,7 +44,7 @@ where
             out_buf.push_back(NewickToken::Start);
         } else {
             out_buf.push_back(NewickToken::End);
-            let mut name: String = node.to_string().split_whitespace().collect();
+            let mut name: String = format!("{}", node);
```

Comment:
I think we should go back to https://github.com/onecodex/taxonomy/pull/43/commits/b1787d0129838745d1feb3f0b2901e14881679b7
I believe the issue is if you have whitespace in the name it might mess up the tree? I'm not sure.

Reply:
I tried adding that back and got this error:

```
error[E0599]: no method named `split_whitespace` found for type parameter `T` in the current scope
  --> src/formats/newick.rs:47:41
   |
47 |             let mut name: String = node.split_whitespace().collect();
   |                                         ^^^^^^^^^^^^^^^^ method not found in `T`
```

This seems to work:

```
let mut name: String = node.to_string().split_whitespace().collect();
```